src/Avalonia.Controls/Window.cs

```diff
@@ -183,6 +183,15 @@ public WindowIcon Icon
             set { SetValue(IconProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the startup location of the window.
+        /// </summary>
+        public WindowStartupLocation WindowStartupLocation
```

Comment:
Oh, apologies! I could swear this change wasn't there last time I looked. I think I'm losing it ;)